wallet/address.go

```diff
@@ -145,9 +151,10 @@ func Key(a Address) AddrKey {
 // created by `Key`.
 // Panics when the `Address` can't be decoded.
 func FromKey(k AddrKey) Address {
-	a, err := DecodeAddress(strings.NewReader(string(k)))
+	a := NewAddress()
+	err := io.Decode(bytes.NewBuffer([]byte(k)), a)
 	if err != nil {
-		panic("Could not decode address in FromKey")
+		panic("Could not decode address in FromKey " + err.Error())
```

Comment:
Separate message from error with colon.

Reply:
Updated in 7ebd50a9743a87e3a8bb063e02111377d4f87241.